wordpress-mobile/WordPress-Android
# Issue/13268 activity type empty view

Parent issue #13268

- Introduces "NoActivities" error state on Activity Log Type filter screen
- Updates message on Activity Log empty screen so it's more generic
- Replaces "rewind" with "restore" in user facing strings

To test:
1. Enable "ActivityLogFiltersFeatureConfig"
2. Turn on airplane mode
3. Open Activity Log
4. Click on "Activity Type" filter
5. Notice "No connection", "Please check your internet connection and retry."
6. Turn off airplane mode
7. Click on "Retry" and notice content is shown
8. Go back to activity log
9. Select a date range for which there are no activities (eg. 1-2 October 2020)
10. Notice "No matching events found", "Try adjusting your date range or activity type filters."
11.  Click on "Activity Type" filter
12. Notice "No activities available", "No activities recorded in the selected date range."

PR submission checklist:
